### PR TITLE
KV Range Scan + Query without index

### PR DESCRIPTION
**Pull request by RichardSmedley**

### CODE REVIEW

File: modules/introduction/partials/new-features-76.adoc [hunk 1]
```diff
@@ -158,6 +158,12 @@ For more information on the vector search, see xref:vector-search:vector-search.
 
 === Data Service
 
+* Couchbase Server 7.6 introduces KV Range Scan.
+A method for accessing documents without necessarily knowing the IDs (keys).
```
Comment (bfavini) [inline modules/introduction/partials/new-features-76.adoc]: KV Range Scan is a method for accessing...

File: modules/introduction/partials/new-features-76.adoc [hunk 2]
```diff
@@ -158,6 +158,12 @@ For more information on the vector search, see xref:vector-search:vector-search.
...
+The most common use case is to get all documents whose IDs start with the same prefix, a `prefixScan`,
```
Comment (bfavini) [inline modules/introduction/partials/new-features-76.adoc]: The most common use case is to retrieve all documents

File: modules/introduction/partials/new-features-76.adoc [hunk 3]
```diff
@@ -158,6 +158,11 @@ For more information on the vector search, see xref:vector-search:vector-search.
 
 === Data Service
 
+* KV Range Scan is a method for accessing documents without necessarily knowing the IDs (keys).
```
Comment (bfavini) [inline modules/introduction/partials/new-features-76.adoc]: @RichardSmedley I chatted with Vishal about this--we iterated on the wording a bit and came up with: 

Introduces KV Range Scan, used  to retrieve all documents in a specified range directly from the Data service. Note that in this initial version, you will achieve better performance using a direct fetch or retrieval from a Query with an Index.

**Review by bfavini — CHANGES_REQUESTED** [review]
See note re: conversation with Vishal

File: modules/introduction/partials/new-features-76.adoc (hunk 3 above)
Comment (RichardSmedley) [reply modules/introduction/partials/new-features-76.adoc]: Updated

**Review by bfavini — APPROVED** [review]

**Pull request merged**